bin/generate-routes.js

```diff
@@ -1,9 +1,8 @@
 import { mkdir, copyFile } from "fs/promises";
 import { join } from "path";
-import { ROUTES } from "../src/routes";
 
 Promise.all(
-  Object.keys(ROUTES).map(async (path) => {
+  ["/daily-auths-report/"].map(async (path) => {
```

Comment:
Not super happy about this, but sometimes ESBuild tries (and fails) to parse the SVG, and in other cases Node doesn't know how to handle the static asset imports.

```
> node -r esbuild-register bin/generate-routes.js

identity-reporting/node_modules/identity-style-guide/dist/assets/img/login-gov-logo.svg:1
<?xml version="1.0" encoding="UTF-8"?>
^

SyntaxError: Unexpected token '<'
```

```
> node -r esbuild-register bin/generate-routes.js

identity-reporting/node_modules/esbuild-register/dist/node.js:2699
        throw error;
        ^

Error: Cannot find module '../node_modules/identity-style-guide/dist/assets/img/login-gov-logo.svg?url'
```

There's probably a way to fix this.